test/OpenTelemetry.Tests/Implementation/Trace/Propagation/TraceContextActivityTest.cs

```diff
@@ -97,9 +94,7 @@ public void TraceContextFormat_IsBlankIfNoHeader()
             var ctx = f.Extract(headers, Getter);
 
             Assert.False(ctx.IsValid());
-
-            // TODO: when ActivityContext supports IsRemote
-            // Assert.True(ctx.IsRemote);
+            Assert.False(ctx.IsRemote);
```

Comment:
wondering why are validating this? if context is not valid, (as expected), why validating anything else on it?

Reply:
ok. i can remove those.

Reply:
just updated.